### DISCUSSION TITLE
Restoring LoadBalancer Service Document

### DISCUSSION THREAD

**Discussion by Vishal-thapa** [vmware-tanzu/velero#3334] [Community support Q&A] [answered]
**What steps did you take and what happened:**
For velero troubleshooting there is mention of loadbalancer service restore and it's limitations [https://velero.io/docs/v1.5/troubleshooting/#known-issue-with-restoring-loadbalancer-service](url). However with kubernetes support for loadbalancer my understanding is after restoring with help of velero any loadbalancer service may or may not change there IP. 

**What did you expect to happen:**
In velero doc there is mention of updating CNAME pointer while restoring the backup if there is . So is there need to update CNAME manually or is there any way we can update the same while running the restore command. 

**Anything else you would like to add:**
Velero version using : 1.5

**Comment by dsu-igeek** [comment, marked as answer]
If your CNAME in DNS is an alias for your load balancer's DNS name then the CNAME entry will need to be updated to whatever the load balancer's name is after the restore.  The load balancer's DNS name is set by Kubernetes and we cannot control it.
